content/zurihac2019/sections/registration.html

```diff
@@ -15,5 +15,57 @@ <h1>Registration</h1>
     come first serve order and let you know.  Please do not book anything before
     we have confirmed a seat for you in a confirmation email.
   </p>
-</section>
 
+  <div class="anchor" id="registration"></div>
+  <h2>Scholarships</h2>
+  <p>
+    Switzerland can be expensive for people visiting from abroad. ZuriHac is a
+    free event, but transportation, lodging, and food might discourage somebody
+    without a good income from attending.  We want to make it easier for anyone
+    to come, and contribute towards making the Haskell community exciting and
+    diverse.
+  </p>
+  <p>
+    For that reason, we will provide a small number of scholarships this year.
+    Because we do not charge anything for tickets, our budget is limited and you
+    can consider this year a "trial run" of the program. Anybody accepted will
+    be supported with an amount between 100.- and 500.- Swiss Francs, depending
+    on your expenses.
+  </p>
+  <p>
+    Our focus is on two groups: people with limited financial means and people
```

Comment:
we can cut the middle "Our focus is on people with limited ... and ...".

That way it's also not "two groups" and subgroups :)